From 78bed88b19aa0732884a0d1904ab25aafdb68b23 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Fri, 19 Jan 2018 15:48:21 +0800
Subject: [PATCH 3231/5242] MLK-17428-9: ARM64: dts: change the clock
 frequency to support 768kHz

commit  215768423b6d30ed5de4083cc1b87ded3cd0c5c1 from
https://source.codeaurora.org/external/imx/linux-imx.git

The bit clock for 786KHz, 16bit, 2 channel is 24.576MHz, and the default
divider of SAI is at least 2, so the minimum master clock should be 49MHz.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Reviewed-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 3c09752..239fc53 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -699,7 +699,7 @@
 	assigned-clocks = <&clk IMX8MQ_CLK_SAI1_SRC>,
 			<&clk IMX8MQ_CLK_SAI1_DIV>;
 	assigned-clock-parents = <&clk IMX8MQ_AUDIO_PLL1_OUT>;
-	assigned-clock-rates = <0>, <24576000>;
+	assigned-clock-rates = <0>, <49152000>;
 	status = "okay";
 };
 
@@ -727,7 +727,7 @@
 	assigned-clocks = <&clk IMX8MQ_CLK_SAI5_SRC>,
 			<&clk IMX8MQ_CLK_SAI5_DIV>;
 	assigned-clock-parents = <&clk IMX8MQ_AUDIO_PLL1_OUT>;
-	assigned-clock-rates = <0>, <24576000>;
+	assigned-clock-rates = <0>, <49152000>;
 	fsl,sai-asynchronous;
 	status = "okay";
 };
-- 
1.7.9.5

